src/app/MessageDef/CommandPath.h

```diff
@@ -44,6 +44,16 @@ enum
     kCsTag_CommandId  = 3,
 };
 
+struct Type
+{
+    chip::EndpointId endpointId;
+    chip::ClusterId clusterId;
+    chip::CommandId commandId;
```

Comment:
What about the rest of the metadata in a command, like SuppressResponse?

What about paths targeting groups?

Since that isn't the focus of this PR, please file a follow-up issue for the two items.